README.md

```diff
@@ -91,9 +91,9 @@ Binary path is `target/release/scaphandre`.
 
 A major pain point in measuring power consumption is doing so inside a virtual machine. A virtual machine usually doesn't have access to power metrics.
 Scaphandre aims at solving that by enabling a communication between a scaphandre instance on the hypervisor host and another one in the virtual machine.
-The scaphandre agent on the host will compute the metrics meaningfull for that virtual machine and the one on the VM exploit those metrics to allow its user to exploit the data as if he had access to power metrics in the first place (as if he was on a bare metal machine).
+The scaphandre agent on the host will compute the metrics meaningful for that virtual machine and the one on the VM accesses those metrics to allow its user to access the data as if they had access to power metrics in the first place (as if they were on a bare metal machine).
```

Comment:
About "he/his -> they/their": I'm not a gender freak, but I _think_ in English the neutral pronoun is indeed what they call the [singular they](https://en.wikipedia.org/wiki/Singular_they). So it feels more "inclusive" but also more grammatically correct. :)